### PR TITLE
[Nokia] Add hwsku.json for the Nokia-7215

### PR DESCRIPTION
**Pull request by carl-nokia**
#### Why I did it
hwsku.json port_type attribute is the preferred method for test to differentiate copper vs sfp+ for Nokia-7215.
#### How I did it
added PR 8370 on which this PR depends first -  then added hwsku.json file according to platform specification. 
#### Which release branch to backport (provide reason below if selected)
Depends on PR 8370 

Needed for Nokia 7215 test pass
- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012
- [ ] 202106

### CODE REVIEW

**Review by yxieca — APPROVED** [review]

File: device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/hwsku.json [hunk 1]
```diff
@@ -0,0 +1,212 @@
+{
+    "interfaces": {
+        "Ethernet0": {
+            "default_brkout_mode": "1x1G",
+            "port_type": "RJ45"
```
Comment (Blueve) [inline device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/hwsku.json:5]: This attribute failed to pass this check during build: https://github.com/Azure/sonic-buildimage/blob/e5c2937faec615367a04075ab1a4dc7aeeee1be2/src/sonic-device-data/tests/hwsku_json_checker

**Review by Blueve — CHANGES_REQUESTED** [review]
Is anywhere consume this attribute? Is yes, you might need update the hwsku_json_checker in your PR as well

**Review by Blueve — APPROVED** [review]

**Review by yxieca — APPROVED** [review]

**Pull request merged**